docs/nodes/nodes-library/core-nodes/DateTime/README.md

```diff
@@ -53,7 +53,22 @@ The start node exists by default when you create a new workflow.
 ### 2. Date & Time node
 
 1. Enter the date that you want to convert in the *Value* field.
-2. Click on the *Add Option* dropdown. 
+2. Click on the *Add Option* dropdown.
 3. Click on the *From Format* option, and enter the format of the input date.
 4. Select the format you want to convert it to from the *To Format* dropdown list.
 5. Click on *Execute Node* to run the workflow.
+
+## FAQs
+
+### What values can I use in the Form Format field?
+
+You can use the following values in the ***Form Format*** field:
+
+| Input | Example | Description |
+|-------|---------|-------------|
+|X | 1616108400 | Unix timestamp |
+|x | 1616108400000 | Unix ms timestamp |
+| DD-MM-YYYY | 19-03-2021 | Date, month, and year separated by `-` |
+| DD/MM/YYYY | 19/03/2021 | Date, month, and year separated by `/` |
+
+You can also use the options available in the ***To Format*** dropdown list.
```

Comment:
Since we are doing a reference table, we should then really add all the possible formats in there.